admin_guide/image_policy.adoc

```diff
@@ -12,13 +12,96 @@ toc::[]
 
 == Overview
 
-You can control which images are allowed to run on your cluster using the ImagePolicy
-admission plug-in (currently considered beta). It allows you to control:
+You can control which images can be imported, tagged and/or run in a cluster.
+There are two facilities for this purpose.
+
+xref:#image-policy-configuring-the-image-policy-admission-plug-in[*Allowed
+Registries for import*] - Is an image policy configuration that allowes to
+restrict image origins to particular set of external registries. This set of
+rules is applied to any image being imported or tagged into any image stream.
+Therefor any image referencing registry not matched by the rule set will be
+rejected.
+
+xref:#image-policy-testing-image-policy-admission-plug-in[*ImagePolicy
+admission plug-in*] - Lets you specify which images are allowed to be run on
+your cluster using. This is a beta feature. It allows you to control:
 
 - *The source of images*: which registries can be used to pull images
-- *Image resolution*: force pods to run with immutable digests to ensure the image does not change due to a re-tag
-- *Container image label restrictions*: force an image to have or not have particular labels
-- *Image annotation restrictions*: force an image in the integrated container registry to have or not have particular annotations
+- *Image resolution*: force pods to run with immutable digests to ensure the
+  image does not change due to a re-tag
+- *Container image label restrictions*: force an image to have or not have
+  particular labels
+- *Image annotation restrictions*: force an image in the integrated container
+  registry to have or not have particular annotations
+
+== Configuring Registires Allowed for Import
+
+Can be achieved in `*_master-config.yaml_*` under
+`imagePolicyConfig:allowedRegistriesForImport` section as demonstrated in the
+following example. By default, the setting is not present, which means, all
+images are allowed.
```

Comment:
that's not true is it?  by default you get the registries we hardcoded as defaults for the whitelist.

Reply:
You're right. It's the ansible-installer that doesn't write this policy into `master-config.yaml`. I totally forgot about `openshift start master --write-config` which generates the setting.

I'll open another PR against openshift-ansible to write the policy by default.

Reply:
I just removed the *default* part.